src/parsers/urdf/model.hxx

```diff
@@ -107,9 +107,9 @@ namespace pinocchio
             model.name = name;
           }
 
-          virtual void addRootJoint (const Inertia& Y, const std::string & body_name)
+          virtual void addRootJoint(const Inertia& Y, const std::string & body_name)
           {
-            addFixedJointAndBody(0, SE3::Identity(), "root_joint", Y, body_name);
+            appendBodyToJoint(0,Y,SE3::Identity(),body_name);
```

Comment:
This is not backward compatible.

Reply:
But this was not the expected behavior and much more a bug I would say.

Reply:
There was no reason to append a joint here.

Reply:
I am not against removing this in the future. However, as it is not backward compatible, I would postpone it to next major release. It can be considered both as a bug and a feature. Some people may rely on this (without even realizing it).

Nonetheless, I find it very convenient, when creating a model, to be sure that there is a frame called root_joint, no matter how the model was build. But I can easily modify my code to create a frame before loading the model.

Reply:
For me this is a bug, we are forcing the creation of a joint while there is no creation of joint asked by the user.